ansible/galaxy.yml

```diff
@@ -0,0 +1,46 @@
+namespace: marinatedconcrete
+name: config
+
+# The version of the collection. Must be compatible with semantic versioning
+version: 1.0.0
```

Comment:
maybe start with 0.1.0 since this is early days for us?

Reply:
I don't know that the version numbers really matter here? We can figure out how to publish this to branches perhaps if we want to make use of them.